spec/factories/relationships.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :relationship do
+    follower_id 1
+    followed_id 1
```

Comment:
Instead of hard-coding ids here, let factory girl create new objects for these:

```
factory :relationship do
  follower
  followed
end
```

Then, set up aliases for your user factory: http://www.rubydoc.info/gems/factory_girl/file/GETTING_STARTED.md#Aliases